Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.h

```diff
@@ -52,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSArray<BUYProductVariant *> *)variantsArray;
 
+/**
+ * Fetch the prices from variants and cache the lowest price on minimumPrice.
+ */
+- (void)updateMinimumPrice;
```

Comment:
Does this need to be exposed? It feels like it should happen lazily, when `minimumPrice` is accessed. I may not have full context though.